net/frr/Makefile

```
@@ -1,5 +1,5 @@
 PLUGIN_NAME=		frr
-PLUGIN_VERSION=		1.22
+PLUGIN_VERSION=		1.23
```

```suggestion
PLUGIN_VERSION=		1.22
```

1.22 is not yet released so we add this on top :)